source/audio/supplier/WavSupplier.cpp

```diff
@@ -0,0 +1,75 @@
+/* WavSupplier.cpp
+Copyright (c) 2025 by tibetiroka
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "WavSupplier.h"
+
+#include "../Sound.h"
+
```

Comment:
I don't remember what we do if `std::` isn't actually used, so either
```suggestion

using namespace std;


```
or
```suggestion


```